pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -908,3 +909,24 @@ func (cr *PerconaServerMongoDB) GetExternalNodes() []*ExternalNode {
 func (cr *PerconaServerMongoDB) MCSEnabled() bool {
 	return mcs.IsAvailable() && cr.Spec.MultiCluster.Enabled
 }
+
+const (
+	FinalizerDeletePVC              = "delete-psmdb-pvc"
+	FinalizerDeletePSMDBPodsInOrder = "delete-psmdb-pods-in-order"
+)
+
+func (cr *PerconaServerMongoDB) GetOrderedFinalizers() []string {
+	order := []string{FinalizerDeletePSMDBPodsInOrder, FinalizerDeletePVC}
+
+	set := sets.NewString(cr.GetFinalizers()...)
+	finalizers := make([]string, 0, len(order))
+
+	for _, f := range order {
+		if !set.Has(f) {
+			continue
+		}
```

Comment:
what if I have another finalizer? it won't be in finalizers list after this method